start-a-node.md

```diff
@@ -19,26 +19,17 @@ $ ./cockroach help start
 
 ## Standard Flags
 
-The `start` command supports the following standard flags, as well as the [advanced flags](#advanced-flags) below and [logging flags](cockroach-commands.html#logging-flags) that can be set on any command.
+The `start` command supports the following standard flags, as well as some [advanced flags](#advanced-flags) and [logging flags](cockroach-commands.html#logging-flags).
 
 Flag | Description
 -----|-----------
 `--certs` | The path to the directory containing the node's [security certificates](create-security-certificates.html). When starting the node with security (i.e., without the `--insecure` flag), the `--certs` flag is required. <br><br> **Default:** certs 
-`--host` | A comma-separated list of addresses at which the node can be reached. This might include the node's internal hostname, internal ip address, external hostname, external ip address, etc. <br><br>**Default:** localhost
+`--host` | The address advertised to other nodes and at which the node can be reached by other nodes and clients. This may be the node's internal hostname, internal ip address, external hostname, external ip address, etc. <br><br>**Default:** Advertise node's hostname and listen on all interfaces
 `--insecure` | Whether or not the cluster is secure (authentication and encrypted client/node and inter-node communication). If the cluster is secure, set the `--certs` flag but leave this flag out. If the cluster is insecure, set this flag.
 `--join` | The address for connecting the node to an existing cluster. When starting the first node, leave this flag out. When starting subsequent nodes, set this flag to the address of any existing node. Optionally, you can specify the addresses of multiple existing nodes as a comma-separated list. 
-`--port` | The port over which the node communicates to the rest of the cluster and clients communicate to the node. <br><br>**Default:** 26257
+`--port` | The port over which the node communicates to other nodes in the cluster and clients communicate to the node. <br><br>**Default:** 26257
```

Comment:
The node doesn't communicate using this port, but other nodes communicate to this node using this port. I think the whole idea of using communication is a bit fraught. Perhaps something like: `The port the node listens on for internal and client communication.`